spring-auto-restdocs-docs/section-snippet.adoc

```diff
@@ -109,4 +113,13 @@ AutoDocumentation.sectionBuilder()
     .build()
 ----
 <1> only selected snippets will be used in the specified order
-<2> if snippets having no content should be skipped from section
+<2> whether snippets having no content should be skipped from section
+
+==== Custom snippet in section
+
+Custom snippets are created by subclassing a link:{restdocs-package}/snippet/Snippet.java[Snippet] interface
+(or more conveniently link:{restdocs-package}/snippet/TemplatedSnippet.java[TemplatedSnippet]). If you want your custom
+ snippet to be included in section and automatically recognised header and include link (file name),
```

Comment:
included in section and automatically recognised header and include link -> included as a section with an automatically recognised header and an include link